docs/programming/profiling-nvidia-gpu-performance.md

```diff
@@ -0,0 +1,1056 @@
+      ```
+
+4. Run `nvidia-smi` 
+
+    - Once logged into the node, you can use the `nvidia-smi` command to monitor GPU usage. 
+    - You can alternatively use:
+    ```
+    $ nvidia-smi -l 2
+    ```
+   where `-l 2` is a flag that will refresh the output every 2 seconds. You can change the interval by adjusting the number (`-l 5` for every 5 seconds). Press `Ctrl+C` to stop the loop.
+
+When you run `nvidia-smi` while an application is executing, the output will display the current processes utilizing the GPU as such:
+
+```
++-----------------------------------------------------------------------------------------+
+| NVIDIA-SMI 570.124.06             Driver Version: 570.124.06     CUDA Version: 12.8     |
+|-----------------------------------------+------------------------+----------------------+
+| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
+|                                         |                        |               MIG M. |
+|=========================================+========================+======================|
+|   0  NVIDIA A100-PCIE-40GB          Off |   00000000:21:00.0 Off |                    0 |
+| N/A   33C    P0             40W /  250W |       1MiB /  40960MiB |     12%      Default |
+|                                         |                        |             Disabled |
++-----------------------------------------+------------------------+----------------------+
+
++-----------------------------------------------------------------------------------------+
+| Processes:                                                                              |
+|  GPU   GI   CI              PID   Type   Process name                        GPU Memory |
+|        ID   ID                                                               Usage      |
+|=========================================================================================|
+|    0    0    0           110318      C    ./vectorAdd                            34MiB  |
++-----------------------------------------------------------------------------------------+
+
+```
+| Column               | Value                                         | 
+| :----------------- | :-------------------------------------------------- | 
+| GPU | In this case, the GPU being used is GPU 0. | 
+| GI/CI	| 0 / 0 — Since MIG (Multi-Instance GPU) is not in use here, both are listed as 0. | 
+| PID	| 110318 — This is the unique Process ID of the application using the GPU. | 
+| Type	| C — This indicates that the process is a Compute process, meaning the GPU is being used for calculations or data processing.| 
+| Process Name	| ./vectorAdd — This is the name of the executable using the GPU. | 
+| GPU Memory Usage	| 34MiB — This shows that the process is using 34 MiB of memory.| 
```

Comment:
Thank you very much for adding an example where there is a process running. This does feel out of place though for me. I think there are two options you can go with here:
1. Keep the example in it's entirety, but make it a subsection under `Getting Started`, and before the section `nvidia-smi on MIG-Enabled GPUs`
2. Modify your `nvidia-smi` output under the `Getting Started` section so that it has a process in it (you would then need to modify the tables accordingly).